lib/common/issues/src/issue.rs

```diff
@@ -4,30 +4,53 @@ use chrono::{DateTime, Utc};
 use schemars::JsonSchema;
 use serde::Serialize;
 
+use crate::dashboard::Code;
 use crate::solution::Solution;
 
-/// Type of the issue code
-pub type CodeType = String;
-
 pub trait Issue {
-    fn code(&self) -> CodeType;
+    /// Differentiates issues of the same type
+    fn distinctive(&self) -> &str;
```

Comment:
I think we need a better name here, `distinctive` is not working out for my brain.

What about `issue_instance_id`?
The idea being that they are class of issues with different particular instances.
WDYT?

Reply:
After thinking about it I could not come up with a better name than `instance_id`, renamed 👍